render/tests/test-hyperscript.js

```diff
@@ -87,7 +87,7 @@ o.spec("hyperscript", function() {
 			o(m("a.x", {
 				className: null
 			}).attrs).deepEquals({
-				className: "x"
+				className: "x null"
```

Comment:
I'm not fond of this behavior - it's very counter-intuitive.

Reply:
I was unsure about how to handle these two cases listed in initial JS Bin (I think they're not listed in main issue):
```
test(m('a.x', {class: null}))
test(m('a.x', {className: null}))
```

It could be revised.